testing/snapcast/APKBUILD

```diff
@@ -38,7 +38,26 @@ build() {
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" installclient installserver
+	make DESTDIR="$pkgdir" installclient installserver || return 1
+}
+
+client() {
+	mkdir -p "$subpkgdir"/usr/bin
+	mv "$pkgdir"/usr/bin/snapclient "$subpkgdir"/usr/bin/snapclient || return 1
+}
+
+server() {
+        install="$pkgname.pre-install"
+	mkdir -p "$subpkgdir"/usr/bin
+	mv "$pkgdir"/usr/bin/snapserver "$subpkgdir"/usr/bin/snapserver || return 1
```

Comment:
When you copy and paste please correct indentation to tabs (we use tabs in apkbuilds)